metagamer/environments/tictactoe.py

```diff
@@ -17,6 +17,28 @@
 logger = logging.getLogger(__name__)
 
 
+def policy_page_lines(board: np.array, player: int = 1) -> tuple:
```

Comment:
question: This doesn't use the player argument, any reason you've chosen to include it?

nitpick: Also for return type use `Tuple[int, int]` (you may need to import it from Typing)

Reply:
Yes, because I think that in any agent more sophisticated than this one, it will be essential to distinguish between an opponent's positions and ones' own.  This policy only looks at what is 'filled' by either player. So I think in general, if we're to build some kind of class or general characteristics of what an agent/policy should look like, it will have to now which moves are it's own. Does this make sense? I know unused arguments are generally bad, but I'm trying to write for the rule, where this policy is probably the exception.

Reply:
I think that argument makes sense and would have been something I'd have said if it wasn't there.

I'd suggest that you don't default to player=1 though, since in most cases the player making the turn is quite important to know and it shouldn't be a given